libs/go/zmscli/dump.go

```diff
@@ -111,6 +119,11 @@ func (cli Zms) displayObjectName(buf *bytes.Buffer, fullResourceName string, obj
 
 func (cli Zms) dumpRole(buf *bytes.Buffer, role zms.Role, auditLog bool, indent1 string, indent2 string) {
 	cli.displayObjectName(buf, string(role.Name), ":role.", indent1)
+	dumpInt32Value(buf, indent2, "member_expiry_days", role.MemberExpiryDays)
+	dumpInt32Value(buf, indent2, "token_expiry_mins", role.TokenExpiryMins)
+	dumpInt32Value(buf, indent2, "cert_expiry_mins", role.CertExpiryMins)
+	dumpBoolValue(buf, indent2, "self_serve", role.SelfServe)
+	dumpStringValue(buf, indent2, "sign_algorithm", string(role.SignAlgorithm))
```

Comment:
should we add role.auditEnabled as well?

Reply:
yup